### ISSUE TITLE
Add Java F3 screen

### ISSUE THREAD

**Issue by YHDiamond** [GeyserMC/Geyser#1858]
**What feature do you want?**
A command like "/geyser f3" which would show all the debug information that you get on Java when you press f3 (preferably in a similar looking GUI).

**Alternatives?**
There are coordinates that can be enabled or disabled, but that doesn't contain things like block looking at, or biome. Theoretically certain options can be added using plugins and be displayed in chat or with a toolbar, but that won't show everything in one place.

**How I would recommend going about adding this** 
@notnotnotswipez and I spoke about this a lot, thinking that a possible way would be to make a custom resource pack with MoLang that would make a request from the Geyser server for information. This pack would be applied on join. Depending on your stored options in Geyser, the request would either return empty, as to not interfere with the Bedrock player's screen, or if they have f3 enabled, the request would contain all the necessary information.

**Comment by EddieDiamondFire** [comment]
Theres a plugin for that https://github.com/Heath123/debuginfo-be

**Comment by YHDiamond** [comment]
Yeah we looked at that, it was even made by a geyser contributor, but we know what to do now to make it a full java f3 screen

**Comment by YHDiamond** [comment]
Thanks for the idea though

**Comment by lukeeey** [comment]
You can either mod the client directly and write some text on the screen (difficult), create a program to overlay the text over the window (not as difficult but still difficult) or mess around with action bars

**Comment by YHDiamond** [comment]
yeah we have an idea of exactly how, notnotnotswipez is researching bedrock behaviour and resource packs to find a way to do it, maybe overwriting the paper doll function (luke you can look in #helpful-chat in the discord)

**Comment by notnotnotswipez** [comment]
Yes, ive figured out how it operates, the main issue is the UI having to be toggled and declare to the server that its been toggled, which is easier said than done considering the best option for the UI modification is the client-side paper doll figure

**Comment by YHDiamond** [comment]
Feature request label please?
